drivers/mm/mm_drv_ti_rat.c

```diff
@@ -100,8 +100,8 @@ void sys_mm_drv_ti_rat_init(struct address_trans_region_config *region_config,
 
 int sys_mm_drv_page_phys_get(void *virt, uintptr_t *phys)
 {
-	if (virt == NULL) {
-		return -EINVAL;
+	if (phys == NULL) {
+		return 0;
```

Comment:
This doesn't seem right. After this change, every check to see if mapping is valid will return as valid mapping, regardless if the mapping actually exists.

Reply:
Sure, added a comment.